Refactor CheckUpload method to incorporate bandwidth limits

DerDhaval opened the pull request:
- Updated the CheckUpload method to retrieve and calculate both storage and bandwidth usage limits for projects.
- Enhanced response handling to include bandwidth usage metrics, ensuring users are notified when they exceed either storage or bandwidth limits.
- Modified the sendResponse function to accommodate new bandwidth parameters, improving clarity in usage notifications.


What: 

Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/main/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/main/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

DerDhaval committed:
Refactor CheckUpload method to incorporate bandwidth limits

- Updated the CheckUpload method to retrieve and calculate both storage and bandwidth usage limits for projects.
- Enhanced response handling to include bandwidth usage metrics, ensuring users are notified when they exceed either storage or bandwidth limits.
- Modified the sendResponse function to accommodate new bandwidth parameters, improving clarity in usage notifications.

princeparmar commented on satellite/console/consoleweb/consoleapi/buckets.go:
```diff
@@ -353,32 +353,41 @@ func (b *Buckets) CheckUpload(w http.ResponseWriter, r *http.Request) {
+	storageLimit := usageLimits.StorageLimit
+	storageUsed := usageLimits.StorageUsed
+	bandwidthLimit := usageLimits.BandwidthLimit
+	bandwidthUsed := usageLimits.BandwidthUsed
```
keep check here for 0

DerDhaval committed:
Refactor CheckUpload method to improve response handling and calculations

- Simplified the CheckUpload method by consolidating the logic for calculating remaining storage and bandwidth.
- Enhanced the handling of zero limits to provide a clearer response when no resources are available.
- Improved the calculation of usage percentages to ensure accuracy and prevent division by zero errors.
